src/edit_session.js

```diff
@@ -187,6 +190,36 @@ class EditSession {
         return this.doc;
     }
 
+    /**
+     * Get "widgetManager" from EditSession
+     * 
+     * @returns {LineWidgets} object
```

Comment:
This gives the `widgetManager` the `LineWidgets` type, which I think means that `session.widgetManager.addLineWidget(...)` will result in a typescript error right? Should this return type be `WidgetManager`?